### PR TITLE
Upgrade hsql to 2.7.1

### PR DESCRIPTION
**Pull request by ylyangtw**
I have tested creating tools and execution in the updated image. All work fine.

### CODE REVIEW

**Comment by navarroc** [comment]
Next time, can you please use the issue number in the branch name? It adds some nice metadata so we can track which issue this was associated with. Thank you!

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<hibernate.jpa.version>1.0.0.Final</hibernate.jpa.version>
 		<hibernate.version>4.3.5.Final</hibernate.version>
-		<hsql.version>2.3.2</hsql.version>
+		<hsql.version>2.7.1</hsql.version>
```
Comment (navarroc) [inline pom.xml:14]: Can we just go to the latest (2.7.3)?

**Review by navarroc — CHANGES_REQUESTED** [review]
Please update to the latest hsqldb.

File: pom.xml (hunk 1 above)
Comment (ylyangtw) [reply pom.xml]: Sure, I just updated it

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 - Kubernetes executor prints exception [#23](https://github.com/ncsa/datawolf/issues/23)
+- Clean dependency updates [#27](https://github.com/ncsa/datawolf/issues/27)
```
Comment (navarroc) [inline CHANGELOG.md:11]: Can you change this entry to something like "hsqldb to 2.7.3" so it's more descriptive of what was done? Thanks! After that, I think it's ready to merge.

Comment (ylyangtw) [reply CHANGELOG.md]: Sure, just did

**Review by navarroc — APPROVED** [review]

**Pull request merged**